chatkit-core/src/test/kotlin/com/pusher/chatkit/rooms/state/DeletedRoomTest.kt

```diff
@@ -0,0 +1,40 @@
+package com.pusher.chatkit.rooms.state
+
+import assertk.assertThat
+import assertk.assertions.isNotNull
+import com.pusher.chatkit.state.ChatkitState
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+
+class DeletedRoomTest : Spek({
+
+    describe("given initial state of one room") {
+        val initialState = ChatkitState(
+                joinedRoomsState = JoinedRoomsState(
+                        mapOf(JoinedRoomsStateTestUtil.roomOneId to JoinedRoomsStateTestUtil.roomOne),
+                        mapOf(JoinedRoomsStateTestUtil.roomOneId to 1)))
+
+        describe("when an event for deleting a room that is part of the state is received") {
+            val updatedState = deletedRoomReducer(initialState,
+                    DeletedRoom(JoinedRoomsStateTestUtil.roomOneId))
+
+            it("then the state should be empty") {
+                assertThat(updatedState.joinedRoomsState).isNotNull().isEmpty()
+            }
+        }
+    }
+
+    describe("given initial empty state=") {
```

Comment:
That's not a real scenario so it's better to remove the test.

Reply:
👍  have also deleted this empty state test in LeftRoomTest & RoomUpdatedTest